Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -958,6 +970,246 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
             SourceIdentifier.arrowSymbol,
         ].compactMap { style.source(withIdentifier: $0) }))
     }
+
+    /**
+     Shows a callout containing the duration of each route.
+     Useful as a way to give the user more information when picking between multiple route alternatives.
+     If the route contains any tolled segments then the callout will specify that as well.
+     */
+    public func showRouteDurationAnnotations(_ routes: [Route]?) {
```

Comment:
An array of routes is not an array of route duration annotations, so the argument should have a label:

```swift
public func showRouteDurations(along routes: [Route]?) {
```